### DISCUSSION TITLE
What does the setting "Assigns node dependant styles to new connectors" do?

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#692] [Help]
Hi all,
pls, see subject.

I originally wanted for a new connector to automatically use the style of its starting node if that node's style has a connector's style attached.
From the sound of it, I thought that the setting "Assigns node dependant styles to new connectors" would do just that, but, alas, it looks to me as if it the setting does nothing.
Any helpful thoughts?

MTIA and regards, a.

**Comment by macmarrum** [comment]
Please have a look at the docs: https://docs.freeplane.org/attic/new-wiki-initiative/understanding-nodes.html#connector-styles

**Reply by autoca** [reply]
Well, that's what I wrote or not?
Maybe it is a bug, because, as I wrote, that setting does not seem to do anything.

**Reply by macmarrum** [reply]
The setting works OK for me.
Please make sure you have a connector defined on a Style Node, i.e. in `Format->Manage Styles->Edit styles`.
If the issue persists, please attach your (sample) mindmap.

**Reply by autoca** [reply]
[test connector styles.zip](https://github.com/freeplane/freeplane/files/9656365/test.connector.styles.zip)

**Reply by macmarrum** [reply]
Yes, all looks good. And it works for me.

**Reply by autoca** [reply]
not for me

**Reply by autoca** [reply]
the connector should have the same style as the blue node and be solid.
However, it has the connector style 'Default' and is dashed.

**Comment by dpolivaev** [comment]
Currently, it works only if you use Insert->Connect (Control+L), but I do not see any reason that it behaves differently if the connector is set by drag-and-drop. Fixed.

**Reply by autoca** [reply]
Thx, that's it.
If I insert it via the menu or hotkey, then the setting works and the expected style is applied to the connector.
If I do it with drag and drop (right mouse button) then it does not work.
Does the word "Fixed" mean that you found the reason for it after all and fixed it?

**Reply by dpolivaev** [reply]
"Fixed" means that I have found no reason to keep the difference and have unified the behavior.

**Reply by autoca** [reply]
Ahh, ok, yess! Thank you very much!

**Reply by macmarrum** [reply]
A good catch, Dimitry :+1:
